src/index.tsx

```diff
@@ -116,34 +122,30 @@ export class Modalize<FlatListItem = any, SectionListItem = any> extends React.C
       isVisible: false,
       showContent: true,
       overlay: new Animated.Value(0),
-      modalHeight,
+      modalHeight: props.adjustToContentHeight ? undefined : modalHeight,
```

Comment:
```suggestion
      modalHeight: props.adjustToContentHeight ? 0 : modalHeight,
```

From what I see, you check all the time (at least from what I saw) `modalHeight || 0` in the following file. Wouldn't be faster to define it at first? and you wouldn't have to do `modalHeight || 0` anywhere else

Reply:
modal height `undefined` on the first mount works as `height: auto`, if we set the height to 0 onLayout will return `height:0` every time